Makefile

```diff
@@ -130,33 +130,37 @@ cephcsi: check-env
 e2e.test: check-env
 	go test $(GO_TAGS) -mod=vendor -c ./e2e
 
+.PHONY: need-container-cmd
```

Comment:
Can this be added to current PHONY at line 137

Reply:
Well, there are many `.PHONY` statements throughout the file. I'll add it where `check-env` is called.